src/access/AccessControl.cpp

```diff
@@ -291,6 +291,81 @@ CHIP_ERROR AccessControl::Check(const SubjectDescriptor & subjectDescriptor, con
     return CHIP_ERROR_ACCESS_DENIED;
 }
 
+bool AccessControl::IsValid(const Entry & entry)
+{
+    AuthMode authMode;
+    FabricIndex fabricIndex;
+    Privilege privilege;
+    size_t subjectCount = 0;
+    size_t targetCount  = 0;
+
+    SuccessOrExit(entry.GetAuthMode(authMode));
+    SuccessOrExit(entry.GetFabricIndex(fabricIndex));
+    SuccessOrExit(entry.GetPrivilege(privilege));
+    SuccessOrExit(entry.GetSubjectCount(subjectCount));
+    SuccessOrExit(entry.GetTargetCount(targetCount));
+
+    if (fabricIndex == kUndefinedFabricIndex)
+    {
+        ChipLogError(DataManagement, "AccessControl: invalid fabric index");
+        return false;
+    }
+
+    // Operational PASE not supported for v1.0.
+    if (authMode != AuthMode::kCase && authMode != AuthMode::kGroup)
+    {
+        ChipLogError(DataManagement, "AccessControl: invalid auth mode");
+        return false;
+    }
+
+    if (privilege == Privilege::kAdminister && authMode != AuthMode::kCase)
```

Comment:
Perhaps add a note for how commissioning PASE sessions sidestep these checks after `AddNOC`?

Reply:
I don't want to explain the whole commissioning process in the validity check function, however the check function is right above this one in the file, and the special handling is commented there. So if one is not blindly trying to edit a function in access control without looking at the whole file (which only really has those two core functions), one should be fine. Anything further requires really understanding the spec, no way of getting around that.